src/codeu/chat/ServerMain.java

```diff
@@ -34,6 +43,8 @@
 final class ServerMain {
 
   private static final Logger.Log LOG = Logger.newLog(ServerMain.class);
+  public static File persistentPath = null;
+  public Queue<String> persistentLog = new LinkedList<String>();
```

Comment:
I think in my most updated commit I changed it to be a LinkedList instantiating a LinkedList since I don't think Queue had a pop method, but I'm not completely sure which is generally good practice.